frontend/app/controllers/MemberOnlyContent.scala

```diff
@@ -1,51 +1,56 @@
 package controllers
 
 import com.gu.contentapi.client.model.v1.{MembershipTier => ContentAccess}
-import com.gu.i18n.CountryGroup.UK
+import com.gu.i18n.CountryGroup._
 import com.netaporter.uri.Uri
 import com.netaporter.uri.dsl._
 import com.typesafe.scalalogging.LazyLogging
-import configuration.{Config, CopyConfig}
+import configuration.Config
 import model._
 import play.api.libs.concurrent.Execution.Implicits.defaultContext
 import play.api.mvc._
 import services.{GuardianContentService, _}
 import views.support.PageInfo
 
+import scala.concurrent.Future
```

Comment:
I am using Future in line https://github.com/guardian/membership-frontend/pull/1497/files/c7f7fb392ec6250abbc80eb3d44b27529677a2ec#diff-ace483bd85f4ff21fdca4993d0be33deR26